Managers/LuaMan.h

```diff
@@ -162,6 +162,22 @@ namespace RTE {
 #pragma endregion
 
 #pragma region File I/O Handling
+		/// <summary>
+		/// Returns an iterator over all the directories in relativeDirectory, which is relative to the working directory.
+		/// Note that a call to this method overwrites any previously returned iterator from DirectoryList() or FileList().
+		/// </summary>
+		/// <param name="relativeDirectory">Directory path relative to the working directory.</param>
+		/// <returns>Directory iterator.</returns>
```

Comment:
Please flesh out the <returns> entry a bit more. Same thing in the FileList method below

Reply:
By entry you mean a description of how it works, or are you referring to the `entry` variable that was inside of the for-loop in the method? If the former, do you want a mention of why it overwrites the previously returned iterator, or what do you suggest I add?

Reply:
Sorry for the unclear comment, I meant the <returns> comment. I just want more than 2 words in there haha.